src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,41 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
+    private static final Injector injector = Injector
+            .getInstance("mate.academy");
+
     public static void main(String[] args) {
-        MovieService movieService = null;
 
+        MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
         System.out.println(movieService.get(fastAndFurious.getId()));
         movieService.getAll().forEach(System.out::println);
+
+        CinemaHallService cinemaHallService = (CinemaHallService) injector
+                .getInstance(CinemaHallService.class);
+        CinemaHall imc = new CinemaHall(250,"IMC");
+        cinemaHallService.add(imc);
+        cinemaHallService.get(imc.getId());
+        cinemaHallService.getAll().forEach(System.out::println);
+
+        MovieSessionService movieSessionService = (MovieSessionService) injector
+                .getInstance(MovieSession.class);
```

Comment:
It seems there is a mistake in the `getInstance` method call. You're passing `MovieSession.class` instead of `MovieSessionService.class`. This would cause a `ClassCastException` at runtime.